# Add string (a href) to indicate ungoogled-chromium build, see also
# https://github.com/Eloston/ungoogled-chromium/issues/659 -> chrome://settings/help
# TODO: add also to chrome/browser/ui/webui/version_ui.cc -> chrome://version/

--- a/chrome/browser/resources/settings/about_page/about_page.html
+++ b/chrome/browser/resources/settings/about_page/about_page_ungoogled-chromium.html
@@ -119,7 +119,9 @@
             </a>
           </span>
 </if>
-          <div class="secondary">$i18n{aboutBrowserVersion}</div>
+          <div class="secondary">$i18n{aboutBrowserVersion}
+            [<a href="https://github.com/Eloston/ungoogled-chromium">ungoogled-chromium</a>]
+          </div>
 <if expr="chromeos">
           <div class="secondary"
               inner-h-t-m-l="[[getUpdateOsSettingsLink_()]]">
